src/main/java/step4/util/InputValidator.java

```diff
@@ -1,32 +1,37 @@
 package step4.util;
 
-import step4.domain.Data;
+import step4.dto.Data;
 
 public class InputValidator {
 
-    private static final int nameLength = 5;
+    private static final String SEPARATOR = ",";
+
+    private InputValidator() {
+    }
+
+    private static final int NAME_LENGTH = 5;
 
     public static Data validate(Data data) {
         // names null check
-        if(data.getNames() == null || data.getNames().length() < 1 ){
+        if (data.getNames() == null || data.getNames().length() < 1) {
             throw new IllegalArgumentException("입력한 자동차 이름이 없습니다.");
         }
 
-        if(data.getCount()  < 1 ){
+        if (data.getCount() < 1) {
             throw new IllegalArgumentException("입력한 숫자가 올바르지 않습니다.");
         }
 
-        String [] names = data.spliteName(",");
-        for(String name : names){
+        String[] names = StringUtil.splite(data.getNames(), SEPARATOR);
+        for (String name : names) {
             nameCheck(name);
         }
 
         return data;
     }
 
-    private static void nameCheck(String name){
-        if(name.length() > nameLength){
-            String message = "자동차 이름이 {0}자 초과하였습니다.".replaceAll("\\{0\\}", String.valueOf(nameLength));
+    private static void nameCheck(String name) {
+        if (name.length() > NAME_LENGTH) {
+            String message = "자동차 이름이 {0}자 초과하였습니다.".replaceAll("\\{0\\}", String.valueOf(NAME_LENGTH));
```

Comment:
```suggestion
        if (name.length() >= NAME_LENGTH) {
            String message = "자동차 이름이 {0}자 초과하였습니다.".replaceAll("\\{0\\}", String.valueOf(NAME_LENGTH));
```

요구사항 중 "자동차 이름은 5자를 초과할 수 없다." 라는 말의 뜻은,
5자 까지는 허용이 되어야 한다는 말입니다.
예시에 나온 crong 과 honux 에 대해 5자를 초과했다고 Exception 이 뜨고 있네요

코드를 너무 잘 작성해주셔서, 마지막으로 요구사항 구현 여부를 하나하나 체크하다가,
사소한 점 하나를 발견했습니다.
수정할 필요는 없고, 저처럼 참 편집증적인 리뷰어가 있구나 하고 웃고 넘어가주시면 될것 같습니다.

<img width="416" alt="스크린샷 2021-03-28 오전 9 23 30" src="https://user-images.githubusercontent.com/38595650/112738572-4d0dea00-8fa7-11eb-830f-2de3b54b3e68.png">